repositories.bzl

```diff
@@ -1,60 +1,54 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-def pandoc_repositories():
-    http_archive(
-        name = "pandoc_bin_linux",
-        build_file_content = """
+PANDOC_VERSION = "2.8.0.1"
```

Comment:
It's great to have the Pandoc version as a define, but you still have to change the `sha256` when updating to a never version of Pandoc as well. 

The `sha256` may be changed to a define as well, but currently I don't see any issue in leaving it as is.

Reply:
I verified an already did. Sorry that the diff doesn't make this clear.